chia/consensus/default_constants.py

```diff
@@ -69,3 +71,24 @@
     # June 2033
     PLOT_FILTER_32_HEIGHT=uint32(20643000),
 )
+
+
+def update_testnet_overrides(network_id: str, overrides: Dict[str, Any]) -> None:
+    if network_id != "testnet10":
+        return
+    # activate softforks immediately on testnet
+    # these numbers are supposed to match initial-config.yaml
```

Comment:
This would be for a separate PR, but we could load the initial config file here and load out the values instead of duplicating them.  Feel free to close this conversation after reading it.

Reply:
I;ve read it. Thanks for the comment!